src/modules/timezones/entities/timezone.entity.ts

```diff
@@ -0,0 +1,16 @@
+import { Entity, Column, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity()
+export class Timezone {
```

Comment:
All classes that require id and metadata(updated_at and created_at) field should extend the base class to keep codebase DRY.

Reply:
done